paras-nft-contract/src/lib.rs

```diff
@@ -614,6 +609,19 @@ impl Contract {
 
         let is_non_mintable = if (copies - decrease_copies.0) == minted_copies {
             token_series.is_mintable = false;
+            token_series.price = None;
+
+            env::log(
+                json!({
+                    "type": "nft_set_series_price",
```

Comment:
setting the price as null: https://explorer.testnet.near.org/transactions/4BqwekUEhneSS7THdskVTfK6tkMpWQBTiNycC5k1Dsng